pkg/controller/pubsub/pubsub_config_controller.go

```diff
@@ -89,16 +92,16 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 		&handler.EnqueueRequestForObject{},
 		predicate.Funcs{
 			CreateFunc: func(e event.CreateEvent) bool {
-				return e.Object.GetNamespace() == util.GetNamespace()
+				return e.Object.GetNamespace() == util.GetNamespace() && e.Object.GetName() == *AuditConnection
```

Comment:
Please check nil before dereferencing
e.g. AuditConnection != nil && e.Object.GetName() == *AuditConnection

Reply:
I rebased and added the nil checks